### PR TITLE
Improvement - stake now page

### LINKED ISSUE #287: Improve response in the values of the Stake Now page
It seems that the values of different tokens are being calculated in sequence. ATOM first, then Terra, Kava... This makes the the page keep waiting and the total value won't be updated if the value of ATOM is not available. The process of requesting different token amounts should be done in parallel. No matter which value return first, the total amount will be updated. ATOM  can return last without problem.

### PR DESCRIPTION
**Pull request by rachelhox**
## Description

closes #287 

## Checklist
- [ ] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

**Comment by rachelhox** [comment]
All networks added and updated, but more info needed for `defaultConverter` function
https://github.com/forbole/official-website/blob/59d2a42fb9ad6825a6ead48b8c69a3e7d89cf3fc/src/screens/stake_now/utils.ts#L32-L47

File: src/screens/stake_now/utils.ts [hunk 1]
```diff
@@ -25,6 +25,27 @@ export const uIrisToIris = defaultConverter(1000000);
...
+
+// Need adjusting the converter
+export const uFetchAIToFetchAI = defaultConverter(1000000000000000000);
+
+// Regen Network not listed on Coingecko yet
+export const uRegenToRegen = defaultConverter(1000000);
+
+export const uBitsongToBitsong = defaultConverter(1000000);
+
+export const uOasisToOasis = defaultConverter(1000000);
```
Comment (kwunyeung) [inline src/screens/stake_now/utils.ts:40]:
```suggestion
export const uOasisToOasis = defaultConverter(1);
```

File: src/screens/stake_now/utils.ts [hunk 2]
```diff
@@ -25,6 +25,27 @@ export const uIrisToIris = defaultConverter(1000000);
...
+
+// need to adjust the converter
+export const uKusamaToKusama = defaultConverter(1000);
```
Comment (kwunyeung) [inline src/screens/stake_now/utils.ts:43]:
```suggestion
export const uKusamaToKusama = defaultConverter(1);
```

File: src/screens/stake_now/utils.ts [hunk 3]
```diff
@@ -25,6 +25,27 @@ export const uIrisToIris = defaultConverter(1000000);
...
+
+export const uFlowToFlow = defaultConverter(1000000);
```
Comment (kwunyeung) [inline src/screens/stake_now/utils.ts:45]:
```suggestion
export const uFlowToFlow = defaultConverter(1);
```

File: src/screens/stake_now/utils.ts [hunk 4]
```diff
@@ -25,6 +25,27 @@ export const uIrisToIris = defaultConverter(1000000);
...
+
+export const uSolanaToSolana = defaultConverter(1000000);
```
Comment (kwunyeung) [inline src/screens/stake_now/utils.ts:47]:
```suggestion
export const uSolanaToSolana = defaultConverter(1);
```

**Review by kwunyeung — COMMENTED** [review]
I have suggested the correct converter value for `ROSE`, `SOL`, `FLOW` and `KSM`.

File: src/screens/stake_now/components/forbole_stakes/config.tsx [hunk 5]
```diff
@@ -1,11 +1,78 @@
+const dataParams = {
+  cosmos: {
```
Comment (kwunyeung) [inline src/screens/stake_now/components/forbole_stakes/config.tsx:2]: Although it works but I'm curious why the denom converter value is not included in the `dataParams`?

Comment (rachelhox) [reply src/screens/stake_now/components/forbole_stakes/config.tsx:2]: The converter function is also used in `getCosmosNetwork`
https://github.com/forbole/official-website/blob/6c16f3032012c63240e854823d4fa9e472b79682/src/screens/stake_now/components/forbole_stakes/hooks.tsx#L281-L316

**Review by kwunyeung — APPROVED** [review]
tested

**Pull request merged**